### DISCUSSION TITLE
Please help

### DISCUSSION THREAD

**Discussion by ge0kas** [jlesage/docker-crashplan-pro#436] [General]
I 've been using your package several years and the backup is huge (about 9TB)... but now it is orphan!
I followed all steps from here youtube.com/watch?v=_8OMN08VQ6A but now in DSM7.2 I can't set it up.
I am stuck in the welcome screen of the container..
Please help me , take me from the hand...
`<html>
<body>

2023/07/05 21:56:34 | stdout | [cont-init   ] 15-install-pkgs.sh: terminated with error 1.
-- | -- | --
2023/07/05 21:56:34 | stdout | [cont-init   ] 15-install-pkgs.sh:     required by: world[0]
2023/07/05 21:56:34 | stdout | [cont-init   ] 15-install-pkgs.sh:   0 (no such package):
2023/07/05 21:56:34 | stdout | [cont-init   ] 15-install-pkgs.sh: ERROR: unable to select packages:
2023/07/05 21:56:33 | stdout | [cont-init   ] 15-install-pkgs.sh: fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/community/x86_64/APKINDEX.tar.gz
2023/07/05 21:56:33 | stdout | [cont-init   ] 15-install-pkgs.sh: fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/main/x86_64/APKINDEX.tar.gz
2023/07/05 21:56:33 | stdout | [cont-init   ] 15-install-pkgs.sh: installing '0'...
2023/07/05 21:56:33 | stdout | [cont-init   ] 15-install-pkgs.sh: installing requested package(s)...
2023/07/05 21:56:33 | stdout | [cont-init   ] 15-install-pkgs.sh: executing...
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-xdg-runtime-dir.sh: terminated successfully.
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-xdg-runtime-dir.sh: executing...
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-x11-unix.sh: terminated successfully.
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-x11-unix.sh: executing...
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-web-data.sh: terminated successfully.
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-web-data.sh: executing...
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-vnc-password.sh: terminated successfully.
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-vnc-password.sh: creating VNC password file from environment variable...
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-vnc-password.sh: executing...
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-set-tmp-dir-perms.sh: terminated successfully.
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-set-tmp-dir-perms.sh: executing...
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-openbox.sh: terminated successfully.
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-openbox.sh: executing...
2023/07/05 21:56:33 | stdout | [cont-init   ] 10-nginx.sh: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-nginx.sh: executing...
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-init-users.sh: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-init-users.sh: executing...
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-fontconfig-cache-dir.sh: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-fontconfig-cache-dir.sh: executing...
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-clean-tmp-dir.sh: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-clean-tmp-dir.sh: executing...
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-clean-logmonitor-states.sh: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-clean-logmonitor-states.sh: executing...
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-cjk-font.sh: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-cjk-font.sh: executing...
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-check-app-niceness.sh: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-check-app-niceness.sh: executing...
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-certs.sh: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-init   ] 10-certs.sh: executing...
2023/07/05 21:56:32 | stdout | [cont-init   ] executing container initialization scripts...
2023/07/05 21:56:32 | stdout | [cont-secrets] container secrets loaded.
2023/07/05 21:56:32 | stdout | [cont-secrets] loading container secrets...
2023/07/05 21:56:32 | stdout | [cont-env    ] container environment variables initialized.
2023/07/05 21:56:32 | stdout | [cont-env    ] XDG_STATE_HOME: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] XDG_RUNTIME_DIR: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] XDG_DATA_HOME: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] XDG_CONFIG_HOME: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] XDG_CACHE_HOME: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] TAKE_CONFIG_OWNERSHIP: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] QT_STYLE_OVERRIDE: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] QT_STYLE_OVERRIDE: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-env    ] QT_STYLE_OVERRIDE: executing...
2023/07/05 21:56:32 | stdout | [cont-env    ] HOME: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] GTK_THEME: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] GTK_THEME: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-env    ] GTK_THEME: executing...
2023/07/05 21:56:32 | stdout | [cont-env    ] GTK2_RC_FILES: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] GTK2_RC_FILES: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-env    ] GTK2_RC_FILES: executing...
2023/07/05 21:56:32 | stdout | [cont-env    ] DOCKER_IMAGE_VERSION: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] DOCKER_IMAGE_PLATFORM: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] DISPLAY: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] DISPLAY: terminated successfully.
2023/07/05 21:56:32 | stdout | [cont-env    ] DISPLAY: executing...
2023/07/05 21:56:32 | stdout | [cont-env    ] APP_VERSION: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] APP_NAME: loading...
2023/07/05 21:56:32 | stdout | [cont-env    ] loading container environment variables...
2023/07/05 21:56:32 | stdout | [init        ] container is starting...

</body>
</html>`

**Comment by ge0kas** [comment]
Nevermind, it worked, I wrongly put 0 where NOVALUE just to let Synology save the open window... but I could hit the - button to hide the variable...

**Comment by jlesage** [comment]
Note that you can remove all environment variables that you didn't set.  See https://github.com/jlesage/docker-crashplan-pro#deployment-considerations.
